setup.py

```diff
@@ -55,6 +56,7 @@ def read(*rnames):
             'zest.releaser',
         ],
         'test': [
+            'Products.Archetypes',
```

Comment:
on a previous pull request you made Archetypes optional; with this you will always load it on tests; I think this is not right.